index.html

```diff
@@ -42,8 +42,14 @@
 <!-- End of Navigation -->
 
 <!-- Banner image-->
-<div>
-	<img src="banner.png" border="0" class="responsive-image" alt="Null">
+<div id="slider">
+  <figure>
+    <img src="image1.jpg" border="0" class="responsive-image" alt="Null">
+    <img src="image2.jpg" border="0" class="responsive-image" alt="Null">
```

Comment:
it's not image2.jpg, it's image2.jpeg

Reply:
sorry that is typing error. we will fix it